tiny-event-sourcing-lib/src/main/kotlin/ru/quipy/core/SeekingForSuitableClassesAggregateRegistry.kt

```diff
@@ -92,6 +93,9 @@ class SeekingForSuitableClassesAggregateRegistry(
 
         // bring it all together
         aggregates.forEach { aggregateClass ->
+            if (aggregateClass == SagaStepAggregate::class) {
```

Comment:
why is it needed? Seems like this class should not be located is scanned packages anyway?

if is is really required, please add the comment to explain why

Reply:
If we run `tiny-event-sourcing-spring-app` in a project with the library, all modules are scanned and `SagaStepAggregate` is automatically registered (because all packages are named ru.quipy). And also `SagaStepAggregate` is manually registered in the configuration. And we get an error on startup